client/src/components/feedback/comment.js

```diff
@@ -14,7 +14,7 @@ class FeedbackComment extends Component {
           <small className="date">
             {
               // eslint-disable-next-line max-len
-            `${item.date.substring(11, 16)} ${item.date.substring(8, 10)}/${item.date.substring(5, 7)}/${item.date.substring(0, 4)}`
+            `${item.date.substring(8, 10)}/${item.date.substring(5, 7)}/${item.date.substring(0, 4)} ${item.date.substring(11, 16)}`
```

Comment:
You can create Date object from `item.date` and create date view from it
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat
Using `substring()` isn't good decision in this case

Reply:
With **[these](https://www.w3schools.com/js/js_date_methods.asp)** methods code would be much more.

Reply:
Maybe, but in that case, you will work with date object and won't use unpredictable strings `2020-04-23T06:58:19.663Z` or `2020-04-23T6:58:19.663Z`

And it will be simpler to update date view. Now you use magic constants `substring(11, 16)` 11? 16?